VariablesHomework/VariablesHomework.cs

```diff
@@ -19,127 +29,120 @@ public void Declaration()
 
             int orderOfOperations = 2;
             orderOfOperations = 2 + 12 * (orderOfOperations * 3);
-            Assert.IsTrue(orderOfOperations == /*Answer here*/); // Make this pass by ONLY filling in the answer
+            Assert.IsTrue(orderOfOperations == 74); // Make this pass by ONLY filling in the answer
         }
 
         [TestMethod]
         public void Ranges()
         {
-            Assert.IsTrue(int.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(int.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MinValue == /*Answer here*/); // Make this pass
+            Assert.IsTrue(int.MaxValue == 2147483647); // Make this pass
+            Assert.IsTrue(int.MinValue == -2147483648); // Make this pass
+            Assert.IsTrue(float.MaxValue == 3.402823466e+38f); // Make this pass
+            Assert.IsTrue(float.MinValue == -3.40282347e+38f); // Make this pass
+            Assert.IsTrue(double.MaxValue == 1.7976931348623158e+308); // Make this pass
+            Assert.IsTrue(double.MinValue == -1.7976931348623158e+308); // Make this pass
         }
 
         [TestMethod]
         public void Overflow()
         {
-            unchecked // ignore this, it simple means im telling the compiler i know what I am doing.
+            unchecked // ignore this, it simply means i'm telling the compiler i know what I am doing. //this is bizarre. it's like this data is conceptually circular to the program.
             {
-                Assert.IsTrue(int.MaxValue + 1 == /*Answer here*/); // Make this pass (you can use a literal number or a symbol)
-                Assert.IsTrue(int.MinValue - 1 == /*Answer here*/); // Make this pass (you can use a literal number or a symbol)
-                Assert.IsTrue(float.MaxValue + 1 == /*Answer here*/); // Make this pass (you can use a literal number or a symbol)
-                Assert.IsTrue(float.MinValue - 1 == /*Answer here*/); // Make this pass (you can use a literal number or a symbol)
-                Assert.IsTrue(double.MaxValue + 1 == /*Answer here*/); // Make this pass (you can use a literal number or a symbol)
-                Assert.IsTrue(double.MinValue - 1 == /*Answer here*/); // Make this pass (you can use a literal number or a symbol)
+                Assert.IsTrue(int.MaxValue + 1 == -2147483648); // Make this pass (you can use a literal number or a symbol)
+                Assert.IsTrue(int.MinValue - 1 == 2147483647); // Make this pass (you can use a literal number or a symbol)
+                Assert.IsTrue(float.MaxValue + 1 == 3.402823466e+38f + 1); // Make this pass (you can use a literal number or a symbol)
```

Comment:
For the floats/doubles here, don't put math in your answer. Just use a single value on each line (whether that's a literal value like `1.2e34` or a symbol like `decimal.MinValue`).